### ISSUE TITLE
[BUG] update/fix initializeArrayWithRange

### ISSUE THREAD

**Issue by kingdavidmartins** [Chalarangelo/30-seconds-of-code#252]
### Expected Snippet Behavior
Behavior should stay consistent. if `start = 0` and `0` gets mapped/initialized I believe `end` should also mapped/initailized like so. If both arguments are inclusive, or inclusive and/or exclusive I think it should say in the description. My assumption however is that the function arguments `start` and `end` are both inclusive 

```js
initializeArrayWithRange(5) //-> [0,1,2,3,4,5]
initializeArrayWithRange(7, 3) //-> [3,4,5,6,7]
```

### Current Snippet Behavior
However that is not the case and when the arrayRange gets initialized only the start is initialized with the array
```js
const initializeArrayWithRange = (end, start = 0) =>
  Array.from({ length: end - start }).map((v, i) => i + start);
// initializeArrayWithRange(5) -> [0,1,2,3,4]
// initializeArrayWithRange(7, 3) -> [3,4,5,6]
```

### Possible Solution
Simple `(end + 1)` fix should do the trick
```js
const initializeArrayWithRange = (end, start = 0) =>
  Array.from({ length: (end + 1) - start }).map((v, i) => i + start);
// initializeArrayWithRange(5) -> [0,1,2,3,4,5]
// initializeArrayWithRange(7, 3) -> [3,4,5,6,7]

```

**Comment by Chalarangelo** [comment]
Ok, so we're making the range end inclusive to deal with a corner case. I'm all for that, I'll merge the PR fixing this right away.

**Comment by Chalarangelo** [comment]
Fixed in #255.
